### DISCUSSION TITLE
HTTP RFCs IETF Standards updated & refactored for HTTP1.1/2/3. Is Hyper up to date?

### DISCUSSION THREAD

**Discussion by JFortun** [hyperium/hyper#2890] [General]
As we can see well redacted on the first section of this[ Cloudflare blog post](https://blog.cloudflare.com/cloudflare-view-http3-usage/), the HTTP standards and definitions have been updated and refactored.
I wanted to ask if Hyperium devs are aware of this and if there is something to update on our side to adapt to this new standard definitions.
Thanks.

**Comment by seanmonstar** [comment]
I was aware of the new RFCs. My understanding is that a large part of the work was separating the semantics from HTTP/1.1 and HTTP/2, and updating the text to state what happens in practice. Still, I have read through the "changes since RFC ####", and I don't believe there's anything that we need to do, but if you know of a particular update I may have missed, it's always welcome to point it out.
